receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -49,6 +49,7 @@ import (
 	"go.opentelemetry.io/collector/component/componenterror"
```

Comment:
Is this still being used somewhere?

Reply:
Yes for the constant errors `ErrAlreadyStarted`, etc. defined in that package